src/passkit.cs

```diff
@@ -1242,8 +1286,8 @@ interface PKSuicaPassProperties
 		bool GreenCarTicketUsed { [Bind ("isGreenCarTicketUsed")] get; }
 
 		[Export ("blacklisted")]
-		[Deprecated (PlatformName.iOS, 14,5, message: "Use 'Blocked' instead.")] // exists in base class
-		[Deprecated (PlatformName.WatchOS, 7,4, message: "Use 'Blocked' instead.")]
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Use 'Blocked' instead.")] // exists in base class
+		[Deprecated (PlatformName.WatchOS, 8, 0, message: "Use 'Blocked' instead.")]
```

Comment:
same, the version should be the one where `Blocked` was added
and it's also missing one for macOS 11,0 and (likely Catalyst)

Reply:
thanks!

just to confirm, are the web docs misleading about this one? clicking on the hyperlink goes to the `blocked` property for pkstoredvaluepassproperties instead of pktransitpassproperties https://developer.apple.com/documentation/passkit/pksuicapassproperties/2545174-blacklisted?language=objc